packages/tictactoe/notes/ttt.md

```diff
@@ -316,63 +311,70 @@ In `/src/core/test-scenarios.ts` we will define a sequence of states, to be sign
-    + '000000000000000000000000aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa' // asAddress
-    + '000000000000000000000000bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb' // bsAddress
-    + '0000000000000000000000000000000000000000000000000000000000000002' // StateType (PreFundSetup, PostFundSetup, Game, Conclude)
-    + '0000000000000000000000000000000000000000000000000000000000000007' // turnNum
-    + '0000000000000000000000000000000000000000000000000000000000000000' // stateCount ?
-    + '0000000000000000000000000000000000000000000000000000000000000004' // aResolution
-    + '0000000000000000000000000000000000000000000000000000000000000006' // bResolution
-    + '0000000000000000000000000000000000000000000000000000000000000002' // [GameAttributes: GamePositionType]
-    + '0000000000000000000000000000000000000000000000000000000000000001' // [GameAttributes: roundBuyIn]
-    + '0000000000000000000000000000000000000000000000000000000000000018' // [GameAttributes: noughts
-    + '0000000000000000000000000000000000000000000000000000000000000101', // [GameAttributes: crosses]
- ```
- This raw data, along with an appropriate signature, could be used to raise a challenge on chain. We must now write functions that will appropriately convert between these two represenations-- on the one hand, TypeScript interfaces containing human-readable properties of the game, and on the other hand long hex strings representing the same information. 
 
+The balances are defined elsewhere in this file, as are the base parameters shared by all of these states. As an example, considered the encoded version of one of these states:
+
+```javascript
+  playing4Hex: '0x' + '0000000000000000000000001111111111111111111111111111111111111111' // libraryAdress
+   + '0000000000000000000000000000000000000000000000000000000000000004' // channelNonce
+   + '0000000000000000000000000000000000000000000000000000000000000002' // number of participants
+   + '000000000000000000000000aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa' // asAddress
+   + '000000000000000000000000bbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb' // bsAddress
+   + '0000000000000000000000000000000000000000000000000000000000000002' // StateType (PreFundSetup, PostFundSetup, Game, Conclude)
+   + '0000000000000000000000000000000000000000000000000000000000000007' // turnNum
+   + '0000000000000000000000000000000000000000000000000000000000000000' // stateCount ?
+   + '0000000000000000000000000000000000000000000000000000000000000004' // aResolution
+   + '0000000000000000000000000000000000000000000000000000000000000006' // bResolution
+   + '0000000000000000000000000000000000000000000000000000000000000002' // [GameAttributes: GamePositionType]
+   + '0000000000000000000000000000000000000000000000000000000000000001' // [GameAttributes: roundBuyIn]
+   + '0000000000000000000000000000000000000000000000000000000000000018' // [GameAttributes: noughts
```

Comment:
Curious, what was the thinking behind encoding x's and o's in separate fields? If my understanding is correct, we could come up with an encoding both in a single field. What are the advantages of having 2 separate fields?

Reply:
Thanks for the explanation, that makes sense.